__tests__/app.test.js

```diff
@@ -42,142 +42,196 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
 describe("GET /api", () => {
   test("200: responds with an object describing all the available endpoints on your API", () => {
-    const endpointsContents = fs.readFile(`${__dirname}/../endpoints.json`)
-    return Promise.all([endpointsContents])
-      .then((endpointsContents) => {
-        const endpointsActual = JSON.parse(endpointsContents)
-        return request(app)
+    const endpointsContents = fs.readFile(`${__dirname}/../endpoints.json`);
+    return Promise.all([endpointsContents]).then((endpointsContents) => {
+      const endpointsActual = JSON.parse(endpointsContents);
+      return request(app)
         .get("/api")
         .expect(200)
         .then((response) => {
-            Object.keys(response.body.endpoints).forEach((endpoint) => {
-                expect(response.body.endpoints[endpoint]).toMatchObject({
-                    description: expect.any(String),
-                    queries: expect.any(Array),
-                    exampleResponse: expect.any(Object)
-                  });
-            })
-            expect(Object.keys(response.body.endpoints).length).toBe(Object.keys(endpointsActual).length)
+          Object.keys(response.body.endpoints).forEach((endpoint) => {
+            expect(response.body.endpoints[endpoint]).toMatchObject({
+              description: expect.any(String),
+              queries: expect.any(Array),
+              exampleResponse: expect.any(Object),
+            });
+          });
+          expect(Object.keys(response.body.endpoints).length).toBe(
+            Object.keys(endpointsActual).length
+          );
```

Comment:
If you left everything as a string, you'd have a nice time. Does this string look like that string?